plugins/linkattacher/src/linkattacher.cc

```diff
@@ -86,5 +86,35 @@ void LinkAttacher::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
     return;
   }
 
+  if(m_parameters.check("STARTUP"))
+  {
+      yInfo() << LogPrefix << "Applying commands at startup";
```

Comment:
We can remove this? The command can be quite confusing.

Reply:
Ok, I have removed it